src/main/java/net/snowflake/client/core/SFBaseSession.java

```diff
@@ -779,10 +782,6 @@ public SFConnectionHandler getSfConnectionHandler() {
 
   public abstract SnowflakeConnectString getSnowflakeConnectionString();
 
-  public abstract int getHttpClientConnectionTimeout();
```

Comment:
Why we need to remove these two methods?

Reply:
We don't need these two APIs in the SFBaseSession class. SFBaseSession class has two child classes: SFSession and StoredProcSFSession (defined in the snowflake repo). HttpClientConnection is specific to the implementation of `SFSession` and does not apply to `StoredProcSFSession`, therefore removing the two APIs from the base class.